### ISSUE TITLE
Ability to halt entry minting, or freeze contract on 0-Day cases.

### ISSUE THREAD

**Issue by mohammedpatla** [securesecrets/shade#98]
Opening this issue as we want to make sure we are able to halt anything if required, like entry minting, conversion minting for any may day cases.

**Comment by FloppyDisck** [comment]
@mohammedpatla  We could make some special governance proposals that skip voting and can only be triggered by the governance contract's admin. These would be limited to very specific actions like halting contract functionality and such

**Comment by mohammedpatla** [comment]
Yes lets bring this up in thr next meeting. We also need ability to update individual contracts if needed too.

**Comment by DrPresident** [comment]
mint can currently be deactivated, but I'm not a huge fan of having any centralized body in control of these things. There is definitely a need for quick action when needed, but if we need that we should have the multi sig address that we control but it should be transparent to the community
